spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/rdd/CassandraRDDSpec.scala

```diff
@@ -1075,4 +1075,147 @@ class CassandraRDDSpec extends SparkCassandraITFlatSpecBase {
+
+    val results = sc
+      .cassandraTable[(Int, Int, String)](ks, "delete_wide_rows3")
+      .select("key", "group", "value")
+      .collect()
+
+    results should have size 3
+
+    results should contain inOrder(
+      (10, 10, "1010"),
+      (10, 11, "1011"),
+      (10, 12, "1012"))
+
+  }
+
+  it should "delete rows with composite key" in {
+
+    conn.withSessionDo { session =>
+      session.execute(s"""DROP TABLE IF EXISTS $ks.delete_wide_rows4""")
+      session.execute(s"""CREATE TABLE $ks.delete_wide_rows4(key INT, group TEXT, group2 INT, value VARCHAR, PRIMARY KEY ((key, group), group2))""")
+      session.execute(s"""INSERT INTO $ks.delete_wide_rows4(key, group, group2, value) VALUES (10, '1', 1, '1010')""")
+      session.execute(s"""INSERT INTO $ks.delete_wide_rows4(key, group, group2, value) VALUES (10, '1', 2, '1011')""")
+      session.execute(s"""INSERT INTO $ks.delete_wide_rows4(key, group, group2, value) VALUES (10, '2', 1, '1012')""")
+      session.execute(s"""INSERT INTO $ks.delete_wide_rows4(key, group, group2, value) VALUES (1, '2', 2, '2020')""")
+      session.execute(s"""INSERT INTO $ks.delete_wide_rows4(key, group, group2, value) VALUES (2, '2', 1, '2021')""")
+    }
+
+    sc.parallelize(Seq((10, "1", 1), (10, "2", 1)))
+      .deleteFromCassandra(ks, "delete_wide_rows4")
+
+    val results = sc
+      .cassandraTable[(Int, String, Int, String)](ks, "delete_wide_rows4")
+      .select("key", "group", "group2", "value")
+      .collect()
+
+    results should have size 3
+  }
+
+  it should "delete just selected column from the C*" in {
+
+    conn.withSessionDo { session =>
+      session.execute(s"""DROP TABLE IF EXISTS $ks.delete_short_rows""")
+      session.execute(s"""CREATE TABLE $ks.delete_short_rows(key INT, group INT, value VARCHAR, PRIMARY KEY (key,group))""")
+      session.execute(s"""INSERT INTO $ks.delete_short_rows(key, group, value) VALUES (10, 10, '1010')""")
+      session.execute(s"""INSERT INTO $ks.delete_short_rows(key, group, value) VALUES (10, 11, '1011')""")
+      session.execute(s"""INSERT INTO $ks.delete_short_rows(key, group, value) VALUES (10, 12, '1012')""")
+      session.execute(s"""INSERT INTO $ks.delete_short_rows(key, group, value) VALUES (20, 20, '2020')""")
+      session.execute(s"""INSERT INTO $ks.delete_short_rows(key, group, value) VALUES (20, 21, '2021')""")
+      session.execute(s"""INSERT INTO $ks.delete_short_rows(key, group, value) VALUES (20, 22, '2022')""")
+    }
+
+    sc.cassandraTable(ks, "delete_short_rows").where("key = 20")
+      .deleteFromCassandra(ks, "delete_short_rows", SomeColumns("value"))
+
+    val results = sc.cassandraTable[(Int, Int, Option[String])](ks, "delete_short_rows")
+      .collect()
+
+    results should have size 6
+
+    results should contain theSameElementsAs Seq((10, 10, Some("1010")),
+    (10, 11, Some("1011")),
+    (10, 12, Some("1012")),
+    (20, 20, None),
+    (20, 21, None),
+    (20, 22, None))
+
```

Comment:
If possible could we add in tests to cover the examples present in the docs above. 
https://github.com/datastax/spark-cassandra-connector/blob/df8c19e9bd0faf58c2a41c6ce6ea8644591e7cc0/spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/doc/DocExamples.scala